CMakeLists.txt

```diff
@@ -66,6 +66,14 @@ target_compile_definitions(sphinxsys_core INTERFACE _SILENCE_CXX17_ITERATOR_BASE
 find_package(Eigen3 CONFIG REQUIRED)
 target_link_libraries(sphinxsys_core INTERFACE Eigen3::Eigen)
 
+# ## OCCT
+if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "Windows")
```

Comment:
This add a lot of clutter throughout. Add the opencascade moduls under `modules/` folder instead to not have everything at root

Reply:
> This add a lot of clutter throughout. Add the opencascade moduls under `modules/` folder instead to not have everything at root

Actually this part is used to stop OCCT installation in windows since Github hasn't enough storage to install OCCT

Reply:
Yes, I understand that from the condition. But you should not clutter the root CMakeLists with this stuff. You already have `opencascade_modules/CMakeLists.txt` where you can all OpenCascade related things.